index.src.html

```diff
@@ -2075,37 +2064,37 @@
     3. Return the product of |durationInSeconds| and |microsecondsPerSecond|.
 
 ### Methods ###{#audiodata-methods}
-: <dfn method for=AudioData>
-      copyTo(|destination|, |options|)
-    </dfn>
+: <dfn method for=AudioData>byteLength(|options|)</dfn>
```

Comment:
I've made this a method because I realized it wasn't very useful as an attribute. The idea is to tell folks who want to call copyTo() how big their buffer needs to be. 

This has nice symmetry with what's proposed in the VideoFrame readback PR. Its named videoFrame.allocationSize() there, but we've agreed to change that to byteLength() as well.

Reply:
Not a fan of the name, but the concept is nice and I agree this is better as a method.

`byteLengthForCopy` or something.

Reply:
editors call: lets go back to allocationSize() (same for VF)

Reply:
now named allocationSize()